wagtailmedia/templates/wagtailmedia/embeds/audio_embed.html

```diff
@@ -0,0 +1,10 @@
+{% block audio %}
+	<audio
+		class="wagtailmedia-audio"
+		src="{{file}}" 
+		{% if mute %}muted{% endif %}
+		{% if autoplay %}autoplay{% endif %}
+		{% if loop %}loop{% endif %}
+		controls>
+	</audio>
+{% endblock %}
```

Comment:
This might be a stupid question but this project didn’t have anything like this to define the medias’ output before?